ESMF/GOCART2G_GridComp/SS2G_GridComp/SS2G_GridCompMod.F90

```diff
@@ -842,9 +842,11 @@ subroutine Run2 (GC, import, export, clock, RC)
                              SSFLUXU, SSFLUXV, SSCONC, SSEXTCOEF, SSSCACOEF,    &
                              SSEXTTFM, SSSCATFM ,SSANGSTR, SSAERIDX, NO3nFlag=.false.,__RC__)
  
-do n=1,5
-   if(mapl_am_i_root()) print*,'n = ', n,' : Run2 E SS2G sum(ss00n) = ',sum(SS(:,:,:,n))
-end do
+    if (mapl_am_i_root()) then
+      do n = 1, size(SS, 4)
+        print*,'n = ', n,' : Run2 E SS2G sum(ss00n) = ',sum(SS(:,:,:,n))
```

Comment:
@gmao-esherman and @rmontuoro, this appears to be printing something every time-step. Should we still be doing this?

Reply:
@WilliamJamieson - Print statements like the one you pointed out, if needed, could be turned on/off based on global debug level settings for GOCART. However, it would be more helpful if debug output could be sent to ESMF Log files, as  it will provide information for all PETs, not just root. In any case, the code is not necessary in UFS.

Reply:
There are many prints like this in all of GOCART2G's aerosol components. They are not needed anymore.